website/docs/en/plugins/dev/index.mdx

```diff
@@ -130,6 +130,41 @@ The `name` of official Rsbuild plugins uniformly uses `rsbuild:` as a prefix, fo
 
 [rsbuild-plugin-template](https://github.com/rspack-contrib/rsbuild-plugin-template) is a minimal Rsbuild plugin template repository that you can use as a basis for developing your Rsbuild plugin.
 
+### Environment Plugin
+
+Rsbuild supports building outputs for multiple environments at the same time, and supports [add plugins for specified environment](/guide/advanced/environments#add-plugins-for-specified-environment).
+
+If you want the plugin you develop to support use as an Environment plugin, you need to pay attention to the following points:
+
+1. The environment configuration is not equal to the rsbuild configuration:
+   - Use [environment context](/guide/advanced/environments#environment-context) instead of `getRsbuildConfig` to get environment information.
+   - When modifying a specific environment configuration, use [modifyEnvironmentConfig](/plugins/dev/hooks#modifyenvironmentconfig) instead of [modifyRsbuildConfig](/plugins/dev/hooks#modifyrsbuildconfig) (to avoid affecting other environments).
```

Comment:
```suggestion
   - When modifying the Rsbuild config for a specific environment, prioritize using [modifyEnvironmentConfig](/plugins/dev/hooks#modifyenvironmentconfig) instead of [modifyRsbuildConfig](/plugins/dev/hooks#modifyrsbuildconfig) to avoid affecting other environments.
```